examples/authRequest.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018, Google, LLC.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * Import the GoogleAuth library, and create a new GoogleAuth client.
+ */
+const { auth } = require('google-auth-library');
```

Comment:
Why is the following usage used here:
```js
const { auth } = require('google-auth-library');
...
const client = await auth.getClient();
```
Wouldn't it be necessary to create a `GoogleAuth` object here like in the other code examples?
```js
const { GoogleAuth } = require('google-auth-library');
...
const auth = new GoogleAuth({ ... });
const client = await auth.getClient();
```

Reply:
Nope :)  In the last release, we started exporting a top level `auth` object that is just an instance of `GoogleAuth`.  In most cases - that's fine to use :)